int32 agent2_value

uint16 agent2_posx
uint16 agent2_posy
uint16 agent2_post

uint16 agent2_ballx
uint16 agent2_bally
uint16 agent2_ball_detected

uint8 agent2_pass
uint8 agent2_condition